crates/virtio-queue/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# v0.7.0
+
+## Added
+
+- Implement `Eq` for `queue::Queue` and `state::QueueState`.
```

Comment:
This is already mentioned in 0.6.0. Please remove.

Reply:
Whoops, sorry. Fixed